c_tests/tests/ptr_global.c

```diff
@@ -5,7 +5,8 @@
 //   stderr:
 //     ...
 //     i=25
-//     ptr_global: guard-failure
+//     Stopgap
```

Comment:
Should this be something like `jit-state: stopgap` to be in-line with our other jit state prints?

Reply:
Could do. It doesn't matter too much I guess, since this is going to go as soon as we have the stopgap interpreter.

Reply:
I tend to agree with Edd: it's a good discipline to keep to the convention, even if we only need it temporarily.

Reply:
Forgot the "Fixed in e42675f"